juju/docs
# Add documentation-builder to 1.25 docs

## Done
- Added a build script
- Created the navigation yaml
- Moved the media to the src directory
- Updated links to the images in the docs

## QA
- Run `./build-docs.sh`
- Then `cd build`
- Run a server, for example: `python -m SimpleHTTPServer 8000`
- Go to http://localhost:8000/en/config-aws.html
- Look around and check it all works
